app/uploaders/logo_uploader.rb

```diff
@@ -0,0 +1,49 @@
+class LogoUploader < BaseUploader
+  MAX_FILE_SIZE = 3 # Megabytes
+  EXTENSION_ALLOWLIST = %w[svg png jpg jpeg jpe].freeze
+  IMAGE_TYPE_ALLOWLIST = %i[svg png jpg jpeg jpe].freeze
+  CONTENT_TYPE_ALLOWLIST = %w[image/svg+xml image/png image/jpg image/jpeg].freeze
+
+  def store_dir
+    "uploads/logos/"
+  end
+
+  def extension_allowlist
+    EXTENSION_ALLOWLIST
+  end
+
+  def image_type_whitelist
+    # this is needed by CarrierWave::BombShelter
+    IMAGE_TYPE_ALLOWLIST
+  end
+
+  def size_range
+    1..(MAX_FILE_SIZE.megabytes)
```

Comment:
This goes together with the change suggested above:

```suggestion
    1..MAX_FILE_SIZE
```